core/Dockerfile

```diff
@@ -165,7 +165,7 @@ FROM php-base AS python-build
         cd /var/www/MISP || exit; git submodule update --init --recursive .
 EOF
 
-    RUN <<-EOF
+    RUN bash <<'EOF'
```

Comment:
@cudeso are quotes necessary here? My understanding is that they are not

Reply:
This is getting so random...

Do we know what was the shell expansion that did not work without this workaround?
Asking because we might need to retrofit this workaround to other build steps.

Reply:
Yes, basically the expansion did not work in the for loop (line 180 and further).
Printing out the variables `PYPI_MODULE_NAME_VERSION` before and after the the "for mod in" loop indicated they were set. But within the for loop they were empty.
In that for loop, it affected all `PYPI_` variables.

Reply:
It is indeed very weird/random. Using bash, with escaping the EOF was eventually the only way I found working to get those variables retain their value.

Reply:
Not happy yet, trying to reproduce things locally. How are you building it? Are you using podman-compose?

Reply:
Same with `podman-compose` (skipping the compose provider selection)